salt/modules/kubernetes.py

```diff
@@ -381,19 +381,19 @@ def namespaces(**kwargs):
         _cleanup()
 
 
-def deployments(namespace='default', **kwargs):
+def deployments(namespace='default', label_selector='', **kwargs):
```

Comment:
Same as above. No need to change the API signature here. Please reuse `kwargs` for this purpose.